kafka/adminapi.go

```diff
@@ -1012,11 +1158,7 @@ func (a *AdminClient) cToConsumerGroupDescriptions(
 			C.rd_kafka_ConsumerGroupDescription_state(cGroup))
 
 		cNode := C.rd_kafka_ConsumerGroupDescription_coordinator(cGroup)
-		coordinator := Node{
-			ID:   int(C.rd_kafka_Node_id(cNode)),
-			Host: C.GoString(C.rd_kafka_Node_host(cNode)),
-			Port: int(C.rd_kafka_Node_port(cNode)),
-		}
+		coordinator := a.cToNode(cNode)
```

Comment:
We are adding rack information for coordinator in ConsumerGroupDescription. Shall we add this as well to the Changelog?

Reply:
Added a comment in CHANGELOG, it's a bit more generic